server/services/store/sqlstore/data_migrations.go

```diff
@@ -14,6 +14,11 @@ import (
 )
 
 const (
+	// we group the inserts on batches of 1000 because PostgreSQL
+	// supports a limit of around 64K values (not rows) on an insert
+	// query, so we want to stay safely below.
+	CategoryInsertBatch = 1000
```

Comment:
Is there any reason an admin may want to tweak this number?

Reply:
I don't think there is, being the migration a one time operation the admin is not likely to tweak the value. We tested with 5000 entities per batch and it took a couple more seconds, so sadly more is not always faster